### PR TITLE
Entries for SPLC submission 

### PR DESCRIPTION
**Pull request by SundermannC**

### CODE REVIEW

File: literature.bib [hunk 1]
```diff
@@ -335,6 +335,15 @@ @mastersthesis{Guething23
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2024 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{RGS+:JSS24,
+	title = {{Uvlhub: A Feature Model Data Repository Using Uvl and Open Science Principles}},
```
Comment (pmbittner) [inline literature.bib:339]: Is this correctly captialized? UVLHub? UVL?

File: literature.bib [hunk 2]
```diff
@@ -1241,6 +1276,67 @@ @proceedings{VaMoS23
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2022 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{SC:AS22,
+  author  = {Sepúlveda, Samuel and Cravero, Ania},
+  title   = {{Reasoning Algorithms on Feature Modeling -- A Systematic Mapping Study}},
+  journal = {Applied Sciences},
```
Comment (pmbittner) [inline literature.bib:1282]: create variable?

Comment (pmbittner) [reply literature.bib:1282]: Since Sebastian's PR every entry has a variable I think.

File: literature.bib [hunk 3]
```diff
@@ -1241,6 +1276,67 @@ @proceedings{VaMoS23
...
+  volume  = {12},
+  year    = 2022
+}
+
+@article{LRI+:WIREs22,
+  author  = {Le Quy, Tai and Roy, Arjun and Iosifidis, Vasileios and Zhang, Wenbin and Ntoutsi, Eirini},
+  title   = {{A Survey On Datasets For Fairness-Aware Machine Learning}},
+  journal = {WIREs Data Mining and Knowledge Discovery},
```
Comment (pmbittner) [inline literature.bib:1290]: .

File: literature.bib [hunk 4]
```diff
@@ -1960,6 +2056,32 @@ @inproceedings{WKR:SPLC22
...
+	title = {{The Power of Literal Equivalence in Model Counting}},
+	author = {Lai, Yong and Meel, Kuldeep S and Yap, Roland HC},
+	booktitle = AAAI,
+	publisher = AAAIpress,
+	address = Palo,
+	volume = {35},
+	number = {5},
+	pages = {3851--3859},
+	year = 2021
+}
```
Comment (pmbittner) [inline literature.bib:2082]: indentation is inconsistent across all entries

File: literature.bib [hunk 5]
```diff
@@ -2760,6 +2897,32 @@ @inproceedings{SL:VaMoS21
...
+	pages      = {11--20},
+	numpages   = {10},
+	location   = {Natal, Brazil},
+	fmslr-tags = {classified by VFB, FM analysis, repo=0, published=3, solver=(NSGA-III, PCA-NSGA-II)}
+}
+
+@article{HFZ+:neurIPS20,
+  title   = {{Open Graph Benchmark: Datasets for Machine Learning on Graphs}},
+  author  = {Hu, Weihua and Fey, Matthias and Zitnik, Marinka and Dong, Yuxiao and Ren, Hongyu and Liu, Bowen and Catasta, Michele and Leskovec, Jure},
+  journal = {Advances in Neural Information Processing Systems (NeurIPS)},
```
Comment (pmbittner) [inline literature.bib:2919]: Maybe we do not need a variable for every journal/booktitle? @skrieter

File: literature.bib [hunk 6]
```diff
@@ -3539,6 +3749,29 @@ @inproceedings{STS:VaMoS20
...
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2019 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{KDO+:SISAP19,
+	author     = {Kitamura, Takashi and Do, Ngoc Thi Bich and Ohsaki, Hitoshi and Fang, Ling and Yatabe, Shunsuke},
+	title      = {{Test-Case Design by Feature Trees}},
+	booktitle  = {Leveraging Applications of Formal Methods, Verification and Validation. Technologies for Mastering Change},
+	year       = 2012,
+	publisher  = Springer,
+	pages      = {458--473},
+	fmslr-tags = {classified by VFB, FM analysis, repo=0, published=0, solver=(ALLSAT, MiniSAT)}
```
Comment (pmbittner) [inline literature.bib:3759]: address missing

File: literature.bib [hunk 7]
```diff
@@ -6657,6 +6916,24 @@ @inproceedings{FMS+:SANER17
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2016 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{GZH+:ECCV16,
+  title        = {{MS-Celeb-1M: A Dataset and Benchmark for Large-Scale Face Recognition}},
+  author       = {Guo, Yandong and Zhang, Lei and Hu, Yuxiao and He, Xiaodong and Gao, Jianfeng},
+  booktitle    = {Computer Vision -- ECCV 2016},
```
Comment (pmbittner) [inline literature.bib:6922]: This conference name looks messed up. ECCV is an abbreviation for a longer title (european conference on computer vision) as long as I remember from my old days at ICG.

File: literature.bib [hunk 8]
```diff
@@ -6657,6 +6916,24 @@ @inproceedings{FMS+:SANER17
...
+  pages        = {87--102},
+  year         = 2016,
+  organization = Springer
+}
+
+@inproceedings{KFB:VSTTE16,
+	title = {{JavaSMT: A Unified Interface for SMT Solvers in Java}},
+	author = {Karpenkov, Egor George and Friedberger, Karlheinz and Beyer, Dirk},
+	booktitle = {Verified Software. Theories, Tools, and Experiments: 8th International Conference (VSTTE)},
```
Comment (pmbittner) [inline literature.bib:6931]: Variable -> This will be inconsistent when using MYshort.

File: literature.bib [hunk 9]
```diff
@@ -8151,6 +8428,18 @@ @article{RTS+:JLAMP16
...
+
+@article{ETV+:CLSS15,
+  title   = {{Evaluating and Comparing Language Workbenches: Existing Results and Benchmarks for the Future}},
+  author  = {Sebastian Erdweg and Tijs {van der Storm} and Markus Völter and Laurence Tratt and Remi Bosman and William R. Cook and Albert Gerritsen and Angelo Hulshout and Steven Kelly and Alex Loh and Gabriël Konat and Pedro J. Molina and Martin Palatnik and Risto Pohjonen and Eugen Schindler and Klemens Schindler and Riccardo Solmi and Vlad Vergu and Eelco Visser and Kevin {van der Vlist} and Guido Wachsmuth and Jimi {van der Woning}},
+  journal = CLSS,
+  volume  = {44},
+  pages   = {24--47},
+  year    = 2015,
+  issn    = {1477-8424},
+  doi     = {https://doi.org/10.1016/j.cl.2015.08.007}
```
Comment (pmbittner) [inline literature.bib:8440]: A doi field usually should not contain the url prefix.

File: literature.bib [hunk 10]
```diff
@@ -9211,6 +9500,19 @@ @inproceedings{UBB+:15
...
+	title        = {{Preprocessing for Propositional Model Counting}},
+	volume       = {28},
+	doi          = {10.1609/aaai.v28i1.9116},
+	number       = {1},
+	journal      = AAAI,
+	publisher = AAAIpress,
+	address = Palo,
+	author       = {Lagniez, Jean-Marie and Marquis, Pierre},
+	year         = 2014
+}
```
Comment (pmbittner) [inline literature.bib:9513]: weird indents

File: literature.bib [hunk 11]
```diff
@@ -10981,6 +11283,19 @@ @inproceedings{SRS:AOSD13
...
+  author    = {Quinton, Cl\'{e}ment and Romero, Daniel and Duchien, Laurence},
+  title     = {{Cardinality-Based Feature Models with Constraints: A Pragmatic Approach}},
+  year      = 2013,
+  publisher = ACM,
+  address   = NY,
+  doi       = {10.1145/2491627.2491638},
+  booktitle = SPLC,
+  pages     = {162--166},
+  location  = {Tokyo, Japan},
+  series    = {SPLC '13}
```
Comment (pmbittner) [inline literature.bib:11296]: remove series

File: literature.bib [hunk 12]
```diff
@@ -10981,6 +11283,19 @@ @inproceedings{SRS:AOSD13
...
+@inproceedings{QRD:SPLC13,
+  author    = {Quinton, Cl\'{e}ment and Romero, Daniel and Duchien, Laurence},
+  title     = {{Cardinality-Based Feature Models with Constraints: A Pragmatic Approach}},
+  year      = 2013,
+  publisher = ACM,
+  address   = NY,
+  doi       = {10.1145/2491627.2491638},
+  booktitle = SPLC,
+  pages     = {162--166},
+  location  = {Tokyo, Japan},
```
Comment (pmbittner) [inline literature.bib:11295]: dont we have a variable for this? There are quite some entries for Tokyo SPLC's

File: literature.bib [hunk 13]
```diff
@@ -15147,6 +15474,22 @@ @inproceedings{ALH+:ICSR11
 	year = 2011
 }
 
+@inproceedings{PNX+:FOSD11,
+	author    = {Passos, Leonardo and Novakovic, Marko and Xiong, Yingfei and Berger, Thorsten and Czarnecki, Krzysztof and W\k{a}sowski, Andrzej},
+	title     = {{A Study of Non-Boolean Constraints in Variability Models of an Embedded Operating System}},
+	year      = {2011},
```
Comment (pmbittner) [inline literature.bib:15480]: remove brackets maybe

File: literature.bib [hunk 14]
```diff
@@ -15147,6 +15474,22 @@ @inproceedings{ALH+:ICSR11
...
+	isbn      = {9781450307895},
+	publisher = ACM,
+	address   = NY,
+	doi       = {10.1145/2019136.2019139},
+	booktitle = FOSD,
+	articleno = {2},
+	numpages  = {8},
+	location  = {Munich, Germany},
```
Comment (pmbittner) [inline literature.bib:15488]: .

**Review by pmbittner — CHANGES_REQUESTED** [review]

File: literature.bib [hunk 15]
```diff
@@ -531,6 +540,21 @@ @inproceedings{HBPF:SPLC23
...
 
+@inproceedings{GDWB:SPLC23,
+	author = {Galindo, Jos\'{e} A. and Dominguez, Antonio J. and White, Jules and Benavides, David},
+	title = {{Large Language Models to Generate Meaningful Feature Model Instances}},
+	publisher = ACM,
+	address = NY,
+	doi = {10.1145/3579027.3608973},
+	booktitle = SPLC,
+	pages = {15--26},
+	numpages = {12},
```
Comment (ekuiter) [inline literature.bib:551]: Thomas: "we typically remove that tag numpages"

File: literature.bib [hunk 16]
```diff
@@ -1241,6 +1276,67 @@ @proceedings{VaMoS23
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2022 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{SC:AS22,
+  author  = {Sepúlveda, Samuel and Cravero, Ania},
+  title   = {{Reasoning Algorithms on Feature Modeling -- A Systematic Mapping Study}},
+  journal = {Applied Sciences},
+  volume  = {12},
```
Comment (ekuiter) [inline literature.bib:1283]: remove braces

File: literature.bib [hunk 17]
```diff
@@ -1241,6 +1276,67 @@ @proceedings{VaMoS23
...
+  year    = 2022
+}
+
+@article{LRI+:WIREs22,
+  author  = {Le Quy, Tai and Roy, Arjun and Iosifidis, Vasileios and Zhang, Wenbin and Ntoutsi, Eirini},
+  title   = {{A Survey On Datasets For Fairness-Aware Machine Learning}},
+  journal = {WIREs Data Mining and Knowledge Discovery},
+  volume  = {12},
+  number  = {3},
```
Comment (ekuiter) [inline literature.bib:1292]: braces

File: literature.bib [hunk 18]
```diff
@@ -1241,6 +1276,67 @@ @proceedings{VaMoS23
...
+  doi     = {https://doi.org/10.1002/widm.1452},
+  year    = 2022
+}
+
+@inproceedings{HGP+:MODEVAR22,
```
Comment (ekuiter) [inline literature.bib:1297]: we already have this one as HGP+:SPLC22. probably both entries should be merged, and MODEVAR is probably more accurate than SPLC

File: literature.bib [hunk 19]
```diff
@@ -2400,6 +2522,21 @@ @inproceedings{HST:SPLC21
...
+  title     = {{A First Prototype of a New Repository for Feature Model Exchange and Knowledge Sharing}},
+  year      = 2021,
+  isbn      = {9781450384704},
+  publisher = ACM,
+  address   = NY,
+  doi       = {10.1145/3461002.3473949},
+  booktitle = MODEVAR,
+  pages     = {80--85},
+  numpages  = {6},
+  keywords  = {feature model repository, characteristics, variability, requirements},
```
Comment (ekuiter) [inline literature.bib:2536]: probably remove keywords

File: literature.bib [hunk 20]
```diff
@@ -4704,6 +4952,17 @@ @article{BCKW:Dagstuhl19
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2018 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{A:SZJ18,
```
Comment (ekuiter) [inline literature.bib:4955]: SZJ?

File: literature.bib [hunk 21]
```diff
@@ -6657,6 +6916,24 @@ @inproceedings{FMS+:SANER17
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2016 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{GZH+:ECCV16,
+  title        = {{MS-Celeb-1M: A Dataset and Benchmark for Large-Scale Face Recognition}},
+  author       = {Guo, Yandong and Zhang, Lei and Hu, Yuxiao and He, Xiaodong and Gao, Jianfeng},
+  booktitle    = {Computer Vision -- ECCV 2016},
+  pages        = {87--102},
+  year         = 2016,
+  organization = Springer
```
Comment (ekuiter) [inline literature.bib:6925]: publisher?

File: literature.bib [hunk 22]
```diff
@@ -6657,6 +6916,24 @@ @inproceedings{FMS+:SANER17
...
+}
+
+@inproceedings{KFB:VSTTE16,
```
Comment (ekuiter) [inline literature.bib:6928]: we already have this entry as KFB:16, although that entry looks kind of messed up. maybe both can be combined

File: literature.bib [hunk 23]
```diff
@@ -6657,6 +6916,24 @@ @inproceedings{FMS+:SANER17
...
+	title = {{JavaSMT: A Unified Interface for SMT Solvers in Java}},
+	author = {Karpenkov, Egor George and Friedberger, Karlheinz and Beyer, Dirk},
+	booktitle = {Verified Software. Theories, Tools, and Experiments: 8th International Conference (VSTTE)},
+	pages={139--148},
+	year = 2016,
+	organization= Springer
```
Comment (ekuiter) [inline literature.bib:6934]: publisher

File: literature.bib [hunk 24]
```diff
@@ -25525,7 +25900,21 @@ @manual{JML:manual
 	tt-tags = {Design by Contract},
 	month = MAY,
 	year = 2013
-} 
+}
+
+@misc{splc24benchmark:repository,
+  author       = {Sundermann et al.},
```
Comment (ekuiter) [inline literature.bib:25906]: et al.?

File: literature.bib [hunk 25]
```diff
@@ -25525,7 +25900,21 @@ @manual{JML:manual
...
+  title        = {{Repository for Feature-Model Dataset}},
+  howpublished = {\url{https://zenodo.org/doi/10.5281/zenodo.11652924}},
+  year         = 2024
+}
+
+@misc{splc24benchmark:replication,
+  author       = {Sundermann et al.},
```
Comment (ekuiter) [inline literature.bib:25913]: see above

**Review by ekuiter — CHANGES_REQUESTED** [review]

File: literature.bib (hunk 1 above)
Comment (SundermannC) [reply literature.bib:339]: Changed

File: literature.bib (hunk 22 above)
Comment (SundermannC) [reply literature.bib:6928]: Could not find KFB:16. Is this already on the main branch?

Comment (ekuiter) [reply literature.bib:6910]: good catch, that's only in my local version

File: literature.bib [hunk 26]
```diff
@@ -335,6 +335,15 @@ @mastersthesis{Guething23
...
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2024 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{RGS+:JSS24,
+	title = {{UVLHub: A Feature Model Data Repository Using UVL and Open Science Principles}},
+	author = {Romero-Organv{\'\i}dez, David and Galindo, Jos{\'e} A and Sundermann, Chico and Horcas, Jose-Miguel and Benavides, David},
+	journal = JSS,
+	publisher = Elsevier,
+	year = 2024,
+	note = {To appear}
+}
```
Comment (skrieter) [inline literature.bib:345]: The "to appear"-entries should be placed before the  %% 2024 %% comment

File: literature.bib [hunk 27]
```diff
@@ -8151,6 +8412,18 @@ @article{RTS+:JLAMP16
...
+
+@article{ETV+:CLSS15,
+	title   = {{Evaluating and Comparing Language Workbenches: Existing Results and Benchmarks for the Future}},
+	author  = {Sebastian Erdweg and Tijs {van der Storm} and Markus Völter and Laurence Tratt and Remi Bosman and William R. Cook and Albert Gerritsen and Angelo Hulshout and Steven Kelly and Alex Loh and Gabriël Konat and Pedro J. Molina and Martin Palatnik and Risto Pohjonen and Eugen Schindler and Klemens Schindler and Riccardo Solmi and Vlad Vergu and Eelco Visser and Kevin {van der Vlist} and Guido Wachsmuth and Jimi {van der Woning}},
+	journal = CLSS,
+	volume  = {44},
+	pages   = {24--47},
+	year    = 2015,
+	issn    = {1477-8424},
+	doi     = {https://doi.org/10.1016/j.cl.2015.08.007}
```
Comment (skrieter) [inline literature.bib:8424]: Remove prefix `https://doi.org/`

File: literature.bib [hunk 28]
```diff
@@ -1241,21 +1275,68 @@ @proceedings{VaMoS23
...
+	year    = 2022
+}
+
+@article{LRI+:WIREs22,
+	author  = {Le Quy, Tai and Roy, Arjun and Iosifidis, Vasileios and Zhang, Wenbin and Ntoutsi, Eirini},
+	title   = {{A Survey On Datasets For Fairness-Aware Machine Learning}},
+	journal = WIREs,
+	volume  = 12,
+	number  = 3,
+	doi     = {https://doi.org/10.1002/widm.1452},
```
Comment (skrieter) [inline literature.bib:1292]: Remove prefix `https://doi.org/`

**Pull request merged**